lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -561,6 +561,60 @@ ospf:
     config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
     config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]
 
+radius_server:
+  hosts:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/'
+    config_set: '<state> radius-server host <ip>'
+    default_value: ''
+
+  auth-port:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host %s.* auth-port (\d+).*$/'
+    config_set: '<state> radius-server host <ip> auth-port <port>'
+    default_value: ''
```

Comment:
@jonnytpuppet Looking at this closer, I think we need to take a step back and look at the overall design for radius-server.  There is some interaction with the global radius properties that we need to consider in how we handle default values.  Also, the netdev type for radius-server has additional properties that I don't see below.  Would you be willing to chat via webex?